content/v1.17/concepts/_index.md

```diff
@@ -0,0 +1,80 @@
+
+The platform users only need to know the details exposed by the platform, like
+`big` or `small` or `US` or `EU`. Platform users don't need to know any details
+about the underlying provider like instance type or region names. 
+
+Crossplane uses multiple core components to manage the various elements of
+building and managing external resources through Kubernetes. 
+
+* [**The Crossplane pods**]({{<ref "./pods">}}) include the core Crossplane pod and
+  Crossplane RBAC manager pod. Together these pods manage all Crossplane
+  components and resources. 
+
+* [**Providers**]({{<ref "./providers">}}) connect Kubernetes to any external
+  provider, like AWS, Azure or GCP. Providers translate Kubernetes native
+  manifests and API calls into external API calls. Providers are responsible for
+  creating, deleting and managing the lifecycle of their resources.
+
+* [**Managed resources**]({{<ref "./managed-resources">}}) are Kubernetes objects
+  representing things the Provider created outside of Kubernetes. Creating a
+  managed resource in Kubernetes requires a Provider to create a resource.
+  Deleting a managed resource requires a Provider to delete the associated
+  external resource.
+
+* [**Compositions**]({{<ref "./compositions">}}) are a template of managed
+  resources. Compositions describe more complex deployments, combining multiple
+  managed resources and any resource customizations, like the size of a database
+  or the cloud provider region.
+
+* [**Composite Resource Definitions**]({{<ref "./composite-resource-definitions">}})
+  represent a custom API, created by platform engineers and consumed by
+  developers or end users. Composite resource definitions use an OpenAPIv3
+  schema to further extend Kubernetes with custom API endpoints, revisions and
+  more. 
+
+* [**Composite Resources**]({{<ref "./composite-resources">}}) represent all the
+  objects created by a user calling the custom API. Every time a user access the
+  custom API Crossplane creates a single Composite Resource and links all
+  the related managed resources to it. 
+
+* [**Claims**]({{<ref "./claims">}}) are like Composite Resources, but exist
+  in a Kubernetes namespace. Every Claim links to a single cluster scoped
+  Composite Resource. Platform users create Claims in their unique namespace,
+  isolating their resources from other teams in other namespaces. 
+
+* [**Composition Functions**]({{<ref "./composition-functions">}}) are custom
+  programs, written your programming language of choice, to apply logic and
+  loops before or after Crossplane creates resources. 
+
+* [**Patches and Transforms**]({{<ref "./patch-and-transform">}}) allow platform
+  engineers to use user inputs to their custom API and change how Crossplane
+  creates resources. Patches and transforms allow for flexible and
+  abstract inputs like `big` or `encrypted` to have specific meanings when
+  creating the actual managed resources.
```

Comment:
Similar comment as https://github.com/crossplane/docs/pull/804#discussion_r1735443979 here too - this shouldn't be in the `master` content nor should it be in the `v1.17` content since it was removed in https://github.com/crossplane/docs/pull/799/files#diff-b256b27bd644a45659738110dfece51b3c4961f7ca2cdf9913524c04b0a54623. This may be pervasive across multiple areas in both `master` and `v1.17` 🤔